src/pages/listing/index.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import './page_listing.less'
+import config from 'src/config/listingPage'
+import Layout from 'src/components/layout/LayoutPure'
+import Preface from 'src/components/sections/preface/listing'
+import MediaPreview from 'src/components/sections/mediaPreview'
+import ListingSuccessful from 'src/components/sections/listing/successful'
+import ListingExchanges from 'src/components/sections/listing/exchanges'
+import ListingClients from 'src/components/sections/listing/clients'
+import ListingExtra from 'src/components/sections/listing/extra'
+import ListingPrices from 'src/components/sections/listing/prices'
+import Reviews from 'src/components/sections/reviews'
+import PastSupportedTokens from 'src/components/sections/pastSupportedTokens/sto'
+import Team from 'src/components/sections/team'
+import Share from 'src/components/sections/shareNow'
+import Footer from 'src/components/sections/footer'
+import ContactForm from 'src/components/contactFormPopup'
+
+const IndexPage = props => {
```

Comment:
({ language  = 'en' }) =>

Reply:
It's a recommendation.
If you don't know about ES6 features, this is a great article: https://zellwk.com/blog/es6/#let-and-const